agent/installer/installer.go

```diff
@@ -52,11 +55,14 @@ func getSupportedRegistry(ob algo.OutputBuilder) registry {
 
 		// BYOH Bundle Repository. Associate bundle with installer
 		linuxDistro := "Ubuntu_20.04.1_x86-64"
-		addBundleInstaller(linuxDistro, "v1.22.3", &algo.Ubuntu20_4K8s1_22{})
+		addBundleInstaller(linuxDistro, k8sVer, &algo.Ubuntu20_4K8s1_22{})
```

Comment:
I think that the K8s version needs to be specific value and not a variable in this case.
At the moment if well call the method with k8sVer = "1.24" it will again use the bundle installer for "1.22" although different steps might be needed.
As we discussed in our meeting, maybe currently the best option is to keep the version hardcoded but to change from "v1.22.3" to "v1.22"

Reply:
This is something that I overlooked. Alex is right. As we are calling the algorithm for Ubuntu 20.04, Kubernetes 1.22 (algo.Ubuntu20_4K8s1_22), we are bound to hardcode both the OS & K8s major and minor versions. Maybe not directly as a string but it has to always match the algorithm version.
Thank you Alex for your remark.

Reply:
The algorithm if I understand correctly is just to indicate the sequence of steps. These need not be different for every OS-K8s combination. Is that correct?

Can we do something like -
`addBundleInstaller(linuxDistro, "v1.22.*", &algo.Ubuntu20_4K8s1_22{})`

So that in future it is easier to support other minor versions, like -
`addBundleInstaller(linuxDistro, "v1.23.*", &algo.Ubuntu20_4K8s1_22{})`
Note that the algo is still `1_22` which I think should be fine.

Reply:
Consider it done. Thanks both Anusha and Alex.